upstream-sha256: 379ed21d3d90ac846b2264c3cac8455bdb6d0d368c42e0dbe14de5061231ceec
actual-sha256: d31b6175cc0704e5dc1900f56b680132b94b820aa1b3e46686e33ca2040ec41d
--- upstream.expected
+++ tolk.actual
@@ -20,5 +20,5 @@
   float val9 = (*(data1_1000+(alu0+900)));
   int alu1 = (lidx1+(gidx0<<1)+(gidx1*20)+(lidx0*10));
   float val10 = (*(data2_100+alu1));
-  *(data0_100+alu1) = (val0+val1+val2+val3+val4+val5+val6+val7+val8+val9+val10);
+  *(data0_100+alu1) = (0.0f+val0+val1+val2+val3+val4+val5+val6+val7+val8+val9+val10);
 }
